### DISCUSSION TITLE
Delete entity

### DISCUSSION THREAD

**Discussion by Amitharitan25** [AutoMapper/AutoMapper.Extensions.OData#210] [General]
Hi There :)
I just wanted to know if there is any support to delete entities using this library?
I wanted to leverage all the sorting this library gives when requesting to delete/update.

Thanks in advance,
Amit

**Comment by BlaiseD** [comment]
This library generates expressions from [ODataQueryOptions](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.odata.query.odataqueryoptions-1?view=odata-aspnetcore-8.0) - I don't see a delete option.  You may want to consider [AutoMapper .Collection](https://github.com/AutoMapper/AutoMapper.Collection).

Or submit a PR with your approach.
